conans/client/manager.py

```diff
@@ -210,16 +213,50 @@ def info(self, reference, current_path, remote=None, options=None, settings=None
                                               info, registry, graph_updates_info,
                                               remote)
 
+    def _read_profile(self, profile_name):
```

Comment:
In the client_cache is implemented the load_profile, but typically we don't perform checks and we don't raise exceptions if profile doesn't exist in client_cache.